app/models/asset.rb

```diff
@@ -281,6 +281,10 @@ def ensure_correct_derivatives_storage_after_change
     end
   end
 
+  def collection_thumbnail?
+    parent&.is_a? Collection
+  end
```

Comment:
I don't love adding this to the model actually... the Asset really doesn't itself behave any differently.... 

If the only place we currently need this is in the Admin::AssetsController... what if we just make it a controller method in that controller (also exposed as a helper)?  it maybe could even be:

```
def asset_is_collection_thumbnail?
  @asset.parent.is_a? Collection
end
helper_method :asset_is_collection_thumbnail?
```

Like that it's about "the" asset, at `@asset` is even baked in?

Also you don't need `&` on `thing.is_a?` cause even `nil` has an `is_a?`, everything has an `is_a?`! But it also doesn't hurt so no big deal.